x/authz/client/cli/tx.go

```diff
@@ -233,6 +240,38 @@ Examples:
 	return cmd
 }
 
+func buildRules(msg string, rulesBytes []byte) ([]*authz.Rule, error) {
+	type internalRules struct {
```

Comment:
This should be global and not just in Client. We should validate this in Grant tx, inside msg-server or ante.
Also, we should have this per msg-type. Not as a single type.

Reply:
These are not internalRules. May be let's call it `AppAuthzRules` or something better